### ISSUE TITLE
gCNV viewer CNV initial coordinates view 

### ISSUE THREAD

**Issue by Glemiret** [broadinstitute/seqr#2078]
gCNV viewer in seqr
Change the default coordinates of the gCNV view when it first loads in seqr to encompass entire CNV +buffer on either side (⅓ of event size) to have a better visualization of the CNV (a more zoomed out view than what is currently displayed).

**Comment by hanars** [comment]
yes my mistake, it looks like that was what was requested in the inital ticket. I will reopen this

**Comment by Glemiret** [comment]
thanks!
